pkg/pillar/agentlog/agentlog_test.go

```diff
@@ -273,9 +273,7 @@ func increaseTotalRandomly(total int64) int64 {
 
 func generateRandomAvgValue() float64 {
 	// Avg is a float of format 0.00 in range [0.00, 100.00]
-	seed := time.Now().UnixNano()
-	r := rand.New(rand.NewSource(seed))
-	return r.Float64() * 100
+	return rand.Float64() * 100
```

Comment:
@christoph-zededa , out of curiosity, which seed rand will use in this case? It is going to use always the same seed or there is an initialization somewhere?

Reply:
I think it is this one here: https://github.com/golang/go/blob/master/src/runtime/rand.go#L61